### PR TITLE
Plasma/testing partition function

### PR DESCRIPTION
**Pull request by aoifeboyle**
Updated/finished off the tests for the partition function properties relevant to the LTE plasma. Almost done with these tests now.

### CODE REVIEW

**Comment by aoifeboyle** [comment]
@wkerzendorf Could you please merge these two PRs if you're happy with them? :)

File: tardis/plasma/properties/property_collections.py [hunk 1]
```diff
@@ -1,5 +1,17 @@
+from tardis.plasma.properties import (BetaRadiation, LevelBoltzmannFactor,
```
Comment (wkerzendorf) [inline tardis/plasma/properties/property_collections.py]: I think you want to take this out here. That was already changed with your last PR.

**Comment by wkerzendorf** [comment]
hi - but wasn't there new stuff in there?

**Comment by wkerzendorf** [comment]
@aoifeboyle

**Comment by aoifeboyle** [comment]
@wkerzendorf Yes, I wanted to untangle it all so I created a new PR with just the new stuff (not the property collections stuff that was already merged). Is that ok? It's number #316.

**Pull request closed without merging**